packages/compass/src/main/window-manager.ts

```diff
@@ -103,8 +103,13 @@ function showConnectWindow(
   };
 
   debug('creating new main window:', windowOpts);
+  const { networkTraffic } = compassApp.getPreferences().getPreferences();
 
   let window: BrowserWindow | null = new BrowserWindow(windowOpts);
+  if (networkTraffic !== true) {
+    // https://github.com/electron/electron/issues/22995
+    window.webContents.session.setSpellCheckerDictionaryDownloadURL('http://127.0.0.1:0/');
```

Comment:
Turned out not to be safebrowsing after all, “only” fetching an English dictionary from Google servers on first run :slightly_smiling_face:

Reply:
Should we just make it a default for all versions? Do we get anything from having this enabled? 🤔

Reply:
I don’t know, it’s probably all hypothetical since we don’t really have human-readable text input fields, other than maybe the intercom surveys?

Reply:
Oh, another thing that comes to mind here, if we disable it conditionally, should we also listen to the option change here in case it was disabled in the application runtime, or download happens only once when the window is spawned and so it doesn't matter afterwards?

Reply:
Yeah, this method is only a setter and does not trigger a download: https://github.com/electron/electron/blob/8c3c0f0b50634bd62c1a4c9a826d8c7aad329c11/patches/chromium/feat_add_support_for_overriding_the_base_spellchecker_download_url.patch#L29-L31

I don’t think there’s an explicit API to trigger a download.

Reply:
Gotcha, thanks for source pointers. I kinda feel like we might just disable it for every page as this feels very unnecessary, but also don't have a strong preference here, so this looks good to me as-is, I'll resolve this one